make_prg/subcommands/from_msa.py

```diff
@@ -52,95 +75,94 @@ def register_parser(subparsers):
         ),
     )
     subparser_msa.add_argument(
-        "-o",
-        "--outdir",
-        dest="output_dir",
-        action="store",
-        default=".",
-        help="Output directory. Default: %(default)s",
-    )
-    subparser_msa.add_argument(
-        "-n",
-        "--prg_name",
-        dest="prg_name",
-        action="store",
-        help="Prg file name. Default: MSA file name",
-    )
-    subparser_msa.add_argument(
-        "-S",
-        "--seqid",
-        help="Sequence identifier to use for the output sequence/PRG. Default is the file name",
-    )
-    subparser_msa.add_argument(
-        "--no_overwrite",
-        dest="no_overwrite",
+        "--output_graphs",
```

Comment:
```suggestion
        "--output-graphs",
```
Can you also please add a short option?

Reply:
Yes, will add a short option

Reply:
Done in https://github.com/iqbal-lab-org/make_prg/pull/35/commits/eee33ad6f266062c34a8f0918504cf0cb4a1793e